src/App/Pages/Accounts/LoginPasswordlessViewModel.cs

```diff
@@ -45,11 +43,9 @@ public LoginPasswordlessViewModel()
 
         public ICommand RejectRequestCommand { get; }
 
-        public FormattedString FingerprintPhraseFormatted => CreateFingerprintPhrase(LoginRequest.FingerprintPhrase);
+        public string LogInAttemptByLabel => string.Format(AppResources.LogInAttemptByXOnY, LoginRequest?.Email, LoginRequest?.Origin);
```

Comment:
If `LoginRequest` is `null` then the string would not make much sense: "Login attempt by on".
So if it can be `null` I'd show a different string or put a loading indicator if it needs to wait to load or show an error or empty if it's just because of the order on the bindings that it's `null` the first time and then it completes the second time the property is changed.

Reply:
it should have value, will make it empty until the data is loaded and property raised.